tensorboard/plugins/debugger/debug_graphs_helper_test.py

```diff
@@ -108,15 +108,18 @@ def testExtractGatedGrpcTensorsFoundGatedGrpcOps(self):
       gated_debug_ops = [
           (item[0], item[2], item[3]) for item in gated_debug_ops]
 
-      # TODO(#1705): TF 2.0 breaks below.
       self.assertIn(('a', 0, 'DebugIdentity'), gated_debug_ops)
-      self.assertIn(('a/read', 0, 'DebugIdentity'), gated_debug_ops)
       self.assertIn(('b', 0, 'DebugIdentity'), gated_debug_ops)
-      self.assertIn(('b/read', 0, 'DebugIdentity'), gated_debug_ops)
       self.assertIn(('c', 0, 'DebugIdentity'), gated_debug_ops)
-      self.assertIn(('c/read', 0, 'DebugIdentity'), gated_debug_ops)
       self.assertIn(('d', 0, 'DebugIdentity'), gated_debug_ops)
-      self.assertIn(('d/read', 0, 'DebugIdentity'), gated_debug_ops)
+
+      if not tensorflow_python_tf2.enabled():
```

Comment:
We’ll shortly stop running tests on TF 1.x, so these will become dead
code. Can we invert the guard and test against the TF 2.x names instead?

Reply:
I removed this chunk of assertions.

Reply:
Okay, that works, if you’re satisfied with the coverage.